src/vcu/inc/state_vcu.h

```diff
@@ -46,14 +49,26 @@ typedef struct {
   bool ScrollSelect;
 } Buttons_T;
 
+typedef struct {
+  int16_t north;
+  int16_t east;
+} IMUVelocity_T;
```

Comment:
It's a good practice to name these state structs by what they contain and not where they get the information from.